src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -261,6 +262,7 @@ protected MaxwellOptionParser buildOptionParser() {
 
 
 		parser.section( "pubsub" );
+		parser.accepts( "pubsub_endpoint", "optionally set pubsub endpoint" ).withRequiredArg();
```

Comment:
ini pubsub_endpoint nya bakalan perlu http scheme atau cuman hostname aja dwi ? bcc: @dwikynator

Reply:
karena under the hood nya maxwell connect ke pubsub via gRPC, jadi format endpointnya, e.g:
```
pubsub.p.googleapis.com:443
```

Reply:
ok noted dwi